filament/src/details/Texture.h

```diff
@@ -100,12 +99,13 @@ class FTexture : public Texture {
 
     // Returns the max number of levels for a texture of given max dimensions
     static inline uint8_t maxLevelCount(uint32_t maxDimension) noexcept {
```

Comment:
Can these `maxLevelCount` functions be made `constexpr`?

Reply:
I don't think so because `ilogbf` isn't `constexpr`.